projects/ngx-formentry/src/components/radio-button/radio.component.html

```diff
@@ -2,6 +2,7 @@
   <div
     *ngFor="let option of options; let i = index"
     class="cds--form-item cds--radio-wrapper"
+    [ngClass]="{'in-line': orientation === 'horizontal'}"
```

Comment:
As mentioned above, I think these styles should be applied to the parent `fieldset` instead of this div, like so:

```html
<fieldset
  class="cds--fieldset"
  [ngClass]="{ inline: orientation === 'horizontal' }"
>